.github/workflows/test.yml

```diff
@@ -39,15 +39,10 @@ jobs:
   build:
     # This build step should be similar to the deploy build, to make sure we actually test
     # the future deployable
-    name: Build the jar on ubuntu
+    name: Build the dask_planner on ubuntu
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Cache local Maven repository
-        uses: actions/cache@v2
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-v1-jdk11-${{ hashFiles('**/pom.xml') }}
       - name: Set up Python
         uses: conda-incubator/setup-miniconda@v2
         with:
```

Comment:
Is this step still necessary? It doesn't look like we're caching the output of the dask_planner build so we could just do this in each test job?

Reply:
I'm not the best with github workflows. While the maven step is not necessary a step that caches the Rust compilation would be

Reply:
Sure, happy to sync on this and figure out what files we would want to store as artifacts across the run

Reply:
removed this.